### PR TITLE
[fabric] Upgrade the channel capability to version 2.0 

### PR DESCRIPTION
**Pull request by himanshulalarya**
**Changelog**
- Updated: Channel capabilities for fabric 2.0 

**Reviewed by**
@sownak @jagpreetsinghsasan 

 

**Linked issue**
#886

### CODE REVIEW

File: platforms/hyperledger-fabric/configuration/roles/create/configtx/templates/configtxinit.tpl [hunk 1]
```diff
@@ -1,5 +1,23 @@
...
+{% if '2.' in network.version %}
+{% if consensus.name == 'kafka' %}
+  Global: &ChannelCapabilities
+    V2_0: true
+  Orderer: &OrdererCapabilities
+    V2_0: true
+  Application: &ApplicationCapabilities
+    V2_0: true
+{% endif %}
+{% if consensus.name == 'raft' %}
```
Comment (sownak) [inline platforms/hyperledger-fabric/configuration/roles/create/configtx/templates/configtxinit.tpl:12]: Why "if" on consensus if the values are same?

File: docs/source/operationalguide.rst [hunk 2]
```diff
@@ -20,5 +20,6 @@ This section defines the pre-requisites installation and steps for setup of a DL
     operations/upgrade_fabricnetwork.md
     operations/adding_new_org_fabric
     operations/indy_verify.md
+    operations/fabric_verify.md
```
Comment (sownak) [inline docs/source/operationalguide.rst:23]: The verify.md file is missing

Comment (himanshulalarya) [reply docs/source/operationalguide.rst:23]: This will be added in another pull request.

Comment (himanshulalarya) [reply docs/source/operationalguide.rst:23]: Seperate request for story 882

Comment (sownak) [reply docs/source/operationalguide.rst:23]: then remove this line and merge into single commit. Everything else is good

**Pull request merged**